docs/pages/schemas.rst

```diff
@@ -33,6 +40,7 @@ Here is a minimal example of a valid JSON schema for an event.
     properties:
       name:
         title: Name
+        level: confidential
```

Comment:
FWIW, I don't know that this is a good example. Since the `name` property is required for all events (right?), I think maybe this should always be unclassified. Maybe a two-field example better illustrates it:

```yaml
name:
  title: Name
  level: unclassified
  description: Name of event
user:
  title: User name
  level: confidential
  description: Name of user who initiated event
```

Reply:
:+1: good point. I'll update this.